### PR TITLE
Handled the empty @depends on tests

### PR DESCRIPTION
**Pull request by didslm**
Added the condition to check if the method in dependency is null and they just specified '@depends' without anything than the test should just skip it and mention that there is no method specified

### CODE REVIEW

File: src/Framework/TestCase.php [hunk 1]
```diff
@@ -1876,6 +1882,23 @@ private function handleDependencies(): bool
...
+    private function markSkippedForNotSpecifyingDependency(): void
+    {
+        $this->status = BaseTestRunner::STATUS_SKIPPED;
+
+        $this->result->startTest($this);
+
+        $this->result->addError(
+            $this,
+            new SkippedTestError(
+                \sprintf('This method is invalid at depends annotation.')
```
Comment (localheinz) [inline src/Framework/TestCase.php]: Perhaps this is better?

```suggestion
                \sprintf('This method has an invalid @depends annotation.')
```

What do you think?

**Review by localheinz — APPROVED** [review]
Apart from the wording, this looks good to me!

👍

File: src/Framework/TestCase.php (hunk 1 above)
Comment (Diarselimi) [reply src/Framework/TestCase.php]: I thought about this but then I thought maybe the @depends keyword can change in the future than we have to change the message too!

File: tests/end-to-end/execution-order/dependencies-isolation.phpt [hunk 2]
```diff
@@ -48,5 +48,8 @@ This test depends on "DependencyFailureTest::testTwo" to pass.
 3) DependencyFailureTest::testFour
 This test depends on "DependencyFailureTest::testOne" to pass.
 
+4) DependencyFailureTest::testHandlesDependsAnnotationWithNoMethodSpecified
+This method is invalid at depends annotation.
```
Comment (epdenouden) [inline tests/end-to-end/execution-order/dependencies-isolation.phpt]: Update this text and it's green :-)

**Comment by epdenouden** [comment]
Didn't the existing `@depends` validator complain about an invalid (a.k.a. missing) parameter already? I'm going to poke around there for the #3519, so now I'm curious...

**Comment by Diarselimi** [comment]
@epdenouden It did complain that it cannot find the method and fail the test ex: if no method specified
``` ClassName:: ```

**Review by epdenouden — APPROVED** [review]

**Pull request merged**